src/bp/ui-studio/src/web/views/FlowBuilder/FlowToaster.tsx

```diff
@@ -0,0 +1,5 @@
+import { Position, Toaster } from '@blueprintjs/core'
+
+export const FlowToaster = Toaster.create({
+  position: Position.TOP
+})
```

Comment:
That's my mistake, I shouldn't have added that toaster thing, Unnecessary boilerplate. Just call it directly ;)